### PR TITLE
feat: add edit scheduler job component

### PR DESCRIPTION
**Pull request by BLasan**
**Description**
Component to edit selected scheduler job completed.

**Related issues and discussion**
#677 

**Screenshots, if any**
Screenshot from 2020-05-13 08-42-18

**Checklist**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@abhaychawla  Please review this

**Comment by karantakalkar** [comment]
@BLasan remove MCs and refactor like edit office form please

**Comment by BLasan** [comment]
@karantakalkar  Refactored and removed MCs. Please have a look

File: src/app/system/system-routing.module.ts [hunk 1]
```diff
@@ -331,11 +333,30 @@ const routes: Routes = [
...
+            },
+          },
+          {
+            path: ':id',
+            data: { title: extract('View Scheduler Job'), routeParamBreadcrumb: 'id' },
+            children: [
+              {
+                path: 'edit',
+                component: EditSchedulerJobComponent,
+                data: { title: extract('Edit Scheduler Job'), routeParamBreadcrumb: false, breadcrumb: 'Edit Scheduler Job' },
```
Comment (karantakalkar) [inline src/app/system/system-routing.module.ts:352]:
```suggestion
                data: { title: extract('Edit Scheduler Job'), routeParamBreadcrumb: false, breadcrumb: 'Edit' },
```

File: src/app/system/manage-scheduler-jobs/edit-scheduler-job/edit-scheduler-job.component.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <button mat-raised-button color="primary" [disabled]="!jobForm.valid">Submit</button>
+        </mat-card-actions>
+  
+      </mat-card-content>
+
+    </form>
+
+  </mat-card>
+
+</div>
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/edit-scheduler-job/edit-scheduler-job.component.html:44]: new line at EOF

File: src/app/system/manage-scheduler-jobs/edit-scheduler-job/edit-scheduler-job.component.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   * Retrieves the selected job data from `resolve`.
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {SystemService} systemService System Service.
+   * @param {Router} router Router for navigation.
+   * @param {ActivatedRoute} route Activated Route.
+   */
+  constructor(private route: ActivatedRoute,
+              private systemService: SystemService,
+              private router: Router,
+              private formBuilder: FormBuilder, ) {
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/edit-scheduler-job/edit-scheduler-job.component.ts:34]:
```suggestion
              private formBuilder: FormBuilder) {
```

**Comment by karantakalkar** [comment]
squash commits

**Comment by karantakalkar** [comment]
update screenshot

**Comment by karantakalkar** [comment]
@BLasan resolve MCs

**Comment by BLasan** [comment]
> @BLasan resolve MCs,update screenshot, also don't mention pr no in commit message from now on.

May I remove the PR no in this ?

**Comment by karantakalkar** [comment]
yes

**Comment by BLasan** [comment]
> yes

Done

**Comment by karantakalkar** [comment]
update screenshot

**Comment by BLasan** [comment]
> update screenshot

Done

**Pull request merged**